homeassistant/components/tuya/translations/select.en.json

```
@@ -48,6 +48,38 @@
             "on": "On",
             "power_off": "Off",
             "power_on": "On"
+        },
+        "tuya__vacuum_cistern": {
+            "low": "Low",
+            "middle": "Middle",
+            "high": "High",
+            "closed": "Closed"
+        },
+        "tuya__vacuum_collection": {
+            "small": "Small",
+            "middle": "Middle",
+            "large": "Large"
+        },
+        "tuya__vacuum_mode": {
+            "standby": "Standby",
+            "random": "Random",
+            "smart": "Smart",
+            "wall_follow": "Wall Follow",
```

Variable vs human text

```suggestion
            "wall_follow": "Follow Wall",
```